packages/auth/src/authClients/nhost.ts

```diff
@@ -0,0 +1,93 @@
+import type { Session, NhostClient, User } from 'nhost-js-sdk'
+
+export type Nhost = NhostClient
+export type NhostUser = User
+
+import { AuthClient } from './'
+
+type NhostProvider = 'google' | 'github' | 'facebook' | 'linkedin'
+export interface AuthClientNhost extends AuthClient {
+  /**
+   * Log In an existing user with email/password or via a OAuth provider
+   * Log In via a OAuth provider also registers the account in case it doesn't exist
+   * @param options.email The user's email address
+   * @param options.password The user's password
+   * @param options.provider One of NhostProvider
```

Comment:
```suggestion
   * @param {NHostProvider} options.provider The authentication provider to use
```

Reply:
I think we could improve the typings here. I'm not 100% sure of this syntax, but maybe this would work: https://jsdoc.app/tags-typedef.html

Reply:
Also my comment for the type is lame.